### DISCUSSION TITLE
incognito v128+

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#1161] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://s-pscripts.github.io/incognito-rises/

only v129 also this sucks
closed discussion

**Comment by S-PScripts** [comment]
WIP

**Comment by KandyKaneMarvel** [comment]
Can I get the name of the flag straight up? chrome://flags/#captive-portal-popup-window doesn't send me to the thingamajig I need

**Reply by S-PScripts** [reply]
captive portal signin uses pop up window [ in chrome://flags ]

**Reply by LieutenantGst** [reply]
could you make a GitHub page? the link to it is blocked for me.

**Reply by S-PScripts** [reply]
It was already in Github Pages. If you're saying that github.io is blocked, you can view the normal Github source for the page here on my repo: https://github.com/s-pscripts/incognito-rises

**Reply by DankFireD** [reply]
> captive portal signin uses pop up window [ in chrome://flags ]

I don't see it there rip

**Reply by HelloIloveGimkit2** [reply]
did you enable "temporarily unexpire flags for M127, M126"

**Reply by HelloIloveGimkit2** [reply]
search both up and reboot then finally search for captive portal thingy

**Reply by DankFireD** [reply]
oh alright will try that next chance i get
